client/state/selectors/is-unlaunched-site.js

```diff
@@ -14,5 +14,5 @@ import getRawSite from 'calypso/state/selectors/get-raw-site';
 export default function isUnlaunchedSite( state, siteId ) {
 	const site = getRawSite( state, siteId );
 
-	return site && site.launch_status && site.launch_status === 'unlaunched';
+	return !! ( site && site.launch_status && site.launch_status === 'unlaunched' );
```

Comment:
`return site?.launch_status === 'unlaunched'` does the same job equally well and passes the improved high-precision tests, too.

Reply:
Agreed, the shorter version looks better 👍  Updated in 649e9aae17ef74c8713c87d02c2202fc0723f681